spec/features/teams_spec.rb

```diff
@@ -7,15 +7,36 @@
     let!(:user) { create :user, :with_team }
 
     describe 'create team' do
-      it 'should create a new team and set the user as admin' do
+      before do
         visit teams_path
         click_link 'Create new team'
-
+      end
+      
+      it 'should create a new team and set the user as admin' do  
         fill_in 'Team Name', with: 'foobar'
         click_button 'Create new team'
 
         expect(user.teams.last.is_admin?(user)).to be_truthy
```

Comment:
You shouldn't write specs like that on feature tests, it's wrong to check side effects using the objects created. Instead, you should verify on the screen why when you fill the field `Team Name` with `foobar` and click on the button `Create new team` the user become an admin of the created team.